core/src/main/resources/schema/app/20180424131246_BUG-101027_parameterized_root_disk_size.sql

```diff
@@ -0,0 +1,9 @@
+-- // “BUG-101027 parameterized root disk size”
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN rootvolumesize INTEGER DEFAULT 50;
```

Comment:
We might have to consider a different approach here. This will block the stack table on cloud.eng

Reply:
Plu not all the existing stacks has 50G root disk because some of the customers already use custom template with custom size.

Reply:
+1 We have to make sure that the CF template does not fail if it does not use the rootvolume variable as custom CFs have this value hardcoded.

Reply:
This value should be part of the template table because this is an instance specific property. Any thoughts @keyki @mhmxs ?

Reply:
Make sense

Reply:
Moved it to the template table